pcsx2/gui/AppConfig.h

```diff
@@ -20,6 +20,8 @@
 #include "CDVD/CDVDaccess.h"
 #include <memory>
 
+const wxString DefaultTitleTemplate = L"Slot: ${slot} | Speed: ${speed} (${vfps}) | ${videomode} | Limiter: ${limiter} | ${gsdx} | ${omodei} | ${cpuusage}";
```

Comment:
Is this variable necessary? It's only used once.

Note: by default const has internal linkage, so every file that directly or indirectly includes AppConfig.h will have its own copy of the variable. The compiler might be able to optimise it away in a release build, but that probably won't happen in a debug build.

Reply:
Yeah, I'll be removing it. I initially had it for a different intention but not needed any more.